conbench/templates/index.html

```diff
@@ -30,12 +30,12 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
             <thead>
                 <tr>
                     <th scope="col">Run creation time</th>
-                    <th scope="col">Author</th>
+                    <th scope="col" class="author-col" style="width: 195px">Commit author</th>
                     <th scope="col">Repository</th>
                     <th scope="col">Hardware</th>
-                    <th scope="col">Commit</th>
-                    <th scope="col">Commit message</th>
-                    <th scope="col">Reason</th>
+                    <th scope="col" class="commit-col" style="width: 65px">Commit</th>
+                    <th scope="col" style="width: 150px">Commit message</th>
```

Comment:
Thinking loud here, as commit message length is variable. Maybe having it fixed to the max content-length might help avoid any overflows.

Reply:
> the max content-length

Interesting thought! We know that this is 2xx characters (can look this up in code -- what we show here is actually a commit message prefix truncated at a max character count length).

In terms of geometrical width, the max width is somewhat unpredictable. Even if it was predictable, it would be "too wide" to default to this. That is, for the longer messages in particular we _want_ the text to automatically wrap.

I thought:
- this column has the somewhat least predictable content width, which is why giving this column a fixed width keeps the overall layout somewhat predictable. 
- it's likely for commit messages to contain multiple short-ish words separated by whitespace. Conservative text wrapping automatically happens, and prevents overflowing in sane cases. 

For more exotic cases we can do ` overflow-wrap: break-word;` which brutally cuts even long words into pieces: https://css-tricks.com/almanac/properties/o/overflow-wrap/

Reply:
> For more exotic cases we can do ` overflow-wrap: break-word;` which brutally cuts even long words into pieces: https://css-tricks.com/almanac/properties/o/overflow-wrap/

Love this, we can definitely revisit(need basis) and iterate :)